tests/__main__.py

```diff
@@ -7,9 +7,10 @@
 os.chdir(REPO_ROOT)
 
 os.putenv(
-    "PATH", "{}:{}".format(os.path.join(REPO_ROOT, "bin"), os.getenv("PATH"))
+    "PATH", ":".join([os.path.join(REPO_ROOT, "bin"), os.getenv("PATH")])
 )
 os.putenv("DVC_HOME", REPO_ROOT)
+os.putenv("DVC_ISATTY", "True")
```

Comment:
I don't like it being set here. This won't work if running `py.test` directly. Please, move it to `tests/conftest.py`. You can look at how `DVC_TEST` env var is set there. Also, please add a comment explaining why we need this defined in tests.